tests/contrib/executors/test_kubernetes_executor.py

```diff
@@ -627,6 +628,13 @@ def test_get_configmaps(self):
         configmaps = worker_config._get_configmaps()
         self.assertListEqual(['configmap_a', 'configmap_b'], configmaps)
 
+    def test_get_labels(self):
+        worker_config = WorkerConfiguration(self.kube_config)
+        labels = worker_config._get_labels({
+            'dag_id': 'dag_id',
+        })
+        self.assertEqual({'my_label': 'label_id', 'dag_id': 'dag_id'}, labels)
```

Comment:
Could you add another check about behaviour of dynamic labels vs static label to make sure we get the right one out.

Reply:
@ashb what do you mean? `my_label` is dynamic label, `dag_id` is static one.

Reply:
or do you mean I need to add test which demonstrate logic? e.g. overriding with same key?

Reply:
yes, overriding with the same key -- i.e testing that we do the right when both dicts contain the same key